### PR TITLE
chore: Point graddle to official GAPIC artifact

### PR DESCRIPTION
**Pull request by ArturoSalazarB16**
This updates our build.gradle to point to the published GAPIC artifacts. On a follow up I will clean up the local projects previously referenced in the build.

### CODE REVIEW

**Comment by bshi** [comment]
lgtm

File: build.gradle [hunk 1]
```diff
@@ -49,9 +56,9 @@ dependencies {
 
   // Add dependencies here
   implementation 'com.google.maps:fleetengine-auth:1.3.3'
-  implementation project(':gapic-google-maps-fleetengine-v1-java')
-  implementation project(':grpc-google-maps-fleetengine-v1-java')
-  implementation project(':proto-google-maps-fleetengine-v1-java')
+
+  implementation 'com.google.api:gax:1.55.0'
```
Comment (ArturoSalazarB16) [inline build.gradle:60]: is it expected that we need to include this dep too?

Comment (ArturoSalazarB16) [reply build.gradle:60]: @bshi

Comment (bshi) [reply build.gradle:60]: This is a known issue (encountered by @shawnlu-g IIRC).  Fine to have this workaround IMO.

**Review by shawnlu-g — APPROVED** [review]

**Review by bshi — CHANGES_REQUESTED** [review]
Should probably also remove all the vendored code in this change as well right?  `*-google-maps-fleetengine-v1-*`

**Comment by shawnlu-g** [comment]
> Should probably also remove all the vendored code in this change as well right? `*-google-maps-fleetengine-v1-*`

Thanks Bo for reminding us on this, was too focused on the gradle file.

**Comment by ArturoSalazarB16** [comment]
Yup, was thinking of doing a follow up PR to remove those, but can do here since it's a pretty small change

**Comment by bshi** [comment]
Yea I ask because submitting atomically will make absolutely sure build is consuming the gradle dependencies and not the vendored dependencies.

**Review by bshi — APPROVED** [review]
LGTM assuming tests pass.

**Comment by ArturoSalazarB16** [comment]
Thanks for the reminder @bshi 
I've amended my commit PTAL when you get a chance @shawnlu-g

**Comment by jpoehnelt** [comment]
@ArturoSalazarB16 go/github to register your github account

**Review by shawnlu-g — APPROVED** [review]
LGTM, thanks!

**Pull request merged**